parser_test.go

```diff
@@ -292,7 +293,7 @@ func (s *testParserSuite) RunRestoreTest(c *C, sourceSQLs, expectSQLs string) {
 		}
 		restoreSQLs += restoreSQL
 	}
-	comment = Commentf("restore %v; expect %v", restoreSQLs, expectSQLs)
+	comment = Commentf("fmtsql %v; expect %v", restoreSQLs, expectSQLs)
```

Comment:
Revert this replacement.

```suggestion
	comment = Commentf("restore %v; expect %v", restoreSQLs, expectSQLs)
```

Reply:
thank you